lib/components/grpc.ts

```diff
@@ -771,10 +771,25 @@ export default function createGrpcAction<Context extends GatewayContext>(
             'x-request-id': requestId,
             'x-gateway-version': VERSION,
         };
+
         if ('protoPath' in config) {
             debugHeaders['x-api-request-protopath'] = config.protoPath;
         }
 
+        if (headers['uber-trace-id']) {
+            debugHeaders['x-uber-trace-id'] = headers['uber-trace-id'];
+        }
```

Comment:
I believe we don't need to do this because this is specific header (`uber-trace-id` is Jaeger naming as I know) which is used in the `@gravity-ui/nodekit` only and not related to `@gravity-ui/gateway` itself. `@gravity-ui/gateway` can be used without `@gravity-ui/nodekit`